chanacceptor/rpcacceptor.go

```diff
@@ -0,0 +1,41 @@
+package chanacceptor
+
+// ChannelAcceptResponse is a struct containing the response from the RPC client
+// which includes whether or not they accept the open channel request and to
+// which PendingChannelID the response is to.
+type ChannelAcceptResponse struct {
```

Comment:
Looks like this is a simple type that now can be defined locally in the rpcserver, where it is used.